docs/parameter_description.rst

```diff
@@ -0,0 +1,158 @@
+=====================
+Parameter description
+=====================
+
+Data Type
+---------
+In combinaison with the parameter "*IDs*", this selects the object \
+from which targets are selected (targets of the given type).
+
+For example, if "*Data Type*" is set to **Project**, and "*ID*" to 789, it will select the **Datasets** \
+inside if "*Target Data Type*"" is set to **Dataset**.
+
+The "*Data Type*" can also be set to **Tag**, in which case the tagged objects of the "*Target Data Type*" \
+are selected. This can be used for a more flexible selection of target object to process.
+
+IDs
+---
+IDs of the object for the "*Data Type*" selection.
+
+Multiple ID separated with a comma can be provided at the same time. In this case, \
+the objects are processed sequentially (if the script fails at the 3rd object, the 1st and 2nd \
+will already be processed).
+
+Target Data Type
+----------------
+The data type to be selected for processing by the script. Default to "*<selected>*" means that the \
+objects specified by "*Data Type*" and "*IDs*" are the ones to be processed. Otherwise, "*Target Data Type*" \
+must correspond to a child object of the selected type (Image are child of Dataset and Project, \
+Well are child of Screen and Plate, ...). The option "*<all (from selected)>*" is used to apply the script \
+on the selected object and its children (dataset and images in the case of a project selected).
+
+
+File Annotation
+---------------
+Parameter for the import from a CSV file. Provided from the "*Choose File*" selection menu, \
+the file will be temporarily uploaded to OMERO for the duration of the script.
+
+The items listed in the CSV are matched to the found target objects. If an item of the CSV correspond to no \
+target object, the item is skipped (as such, target objects coming from different parent objects can be \
+annotated from a single CSV).
+
+The matching of items in the CSV to target objects is done either via IDs or names (IDs have priority over names, \
+see "*Target ID colname*"" and "*Target name colname*"").
+
+IDs of File Annotation already attached to the object can be used instead of "*Choose File*":
+
+.. image:: images/annotation_13_file_annotation_id.png
+
+Namespace
+---------
+In the case of key-value pairs import, the namespace assigned to the created key-value pairs.
+
+For the three other scripts, used to restrict the selection of key-value pairs to that namespace.
+
+When no namespace is provided, matches the namespace ``openmicroscopy.org/omero/client/mapAnnotation``, which \
+makes the key-value pairs editable in OMERO.web (the only namespace that allows that).
+
+Multiple namespace can be provided as a comma separated list.
+
+All namespace can be selected using the * character.
+
+Old Namespace
```

Comment:
``Old`` means here maybe ``existing`` or ``input`` Namespace. It is debatable how ``old`` (in age) these are. Please judge whether or not it is feasible to re-define this term -> if too much work, leave as is imho.